src/database/sqlite/utils.rs

```diff
@@ -229,9 +229,80 @@ fn migrate_v3_to_v4(conn: &mut rusqlite::Connection) -> Result<(), SqliteDbError
     Ok(())
 }
 
+fn migrate_v4_to_v5(
+    conn: &mut rusqlite::Connection,
+    bitcoin_txs: &[bitcoin::Transaction],
+) -> Result<(), SqliteDbError> {
+    db_exec(conn, |db_tx| {
+        db_tx.execute(
+            "
+            CREATE TABLE transactions (
+                id INTEGER PRIMARY KEY NOT NULL,
+                txid BLOB UNIQUE NOT NULL,
+                tx BLOB UNIQUE NOT NULL
+            );",
+            rusqlite::params![],
```

Comment:
This is missing to actually insert the transactions in DB and no single test caught it..